ModuloThree/src/ModuloThree.java

```diff
@@ -0,0 +1,14 @@
+public class ModuloThree {
+
+    public boolean isDividedByThee(int number){
+       boolean isDivided = true;
+
+       try{ if (number % 3 !=0) throw new IllegalArgumentException();
```

Comment:
Proszę używaj zawsze CTRL+ALT+L aby formatować kod, bo teraz to jest nieczytelne
https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html